Backend/src/db/getters.py

```diff
@@ -26,16 +102,19 @@ def get_all_nodes_from(collection: str, ConnectionClass=MongoConnection) -> list
         documents = db[collection].find()
         nodes = []
         for doc in documents:
-            nodes.extend([Node(**data_item) for data_item in doc['data']])        
+            nodes.extend([Node(**data_item) for data_item in doc["data"]])
     return nodes
 
-def get_all_labels_from(collection: str, ConnectionClass=MongoConnection) -> list[dict]:
+
+def get_all_labels_from(collection: str, ConnectionClass=MongoConnection) -> list:
     """
-    Get list document with labels from a collection. 
+    Get list document with labels from a collection.
     """
     with ConnectionClass() as (_, db):
-        labels = list(db[collection].find({}, {"about.labels" : 1, "_id": 0}))
-        return labels
+        result = db[collection].find_one({}, {"about.labels": 1, "_id": 0})
+        if result is None:
+            raise ValueError(f"Error: Could not find any labels in {collection}")
+        return result["about"]["labels"]
```

Comment:
The get labels endpoint results in 500 response atm, Think the return statement has to be changed to `results["about"]["labels"]["coarse"]` we might need to make it more dynamic in the future but I think its ok for now to just hard code it since we are currently only working with one source dataset.

Reply:
# Notes for future
We talked about this, we will only use ["about"]["labels"] for now